LICENSE

```diff
@@ -0,0 +1,25 @@
+Copyright (c) IBM Corp. 2014,2016. All Rights Reserved.
```

Comment:
Yes, this is the license for this module, updating using itself (via a wrapper).